crates/client/src/client.rs

```diff
@@ -508,6 +511,28 @@ impl FuelClient {
             .collect()
     }
 
+    /// Get storage read replay for a block
+    pub async fn storage_read_replay(
+        &self,
+        height: &BlockHeight,
+    ) -> io::Result<Vec<StorageReadReplayEvent>> {
+        let query: Operation<
+            schema::storage_read_replay::StorageReadReplay,
+            schema::storage_read_replay::StorageReadReplayArgs,
+        > = schema::storage_read_replay::StorageReadReplay::build(
```

Comment:
nit: Does this need the type annotation? I feel like all the info is in the construction.

Reply:
No. I cleaned it up.